CONTRIBUTING.md

```diff
@@ -16,7 +16,7 @@ We welcome Pull Requests for bug fixes on any open issues that you'd like to con
 
 ## Discussion
 
-Aside from writing issues and pull requests, you can also hang out in the #mobile WordPress.org Slack channel with other WordPress Mobile developers/contributors. Check out chat.wordpress.org for instructions on how to create an account in Slack.
+Aside from writing issues and pull requests, you can also hang out in the #mobile WordPress.org Slack channel with other WordPress Mobile developers/contributors. Check out [here](chat.wordpress.org) for instructions on how to create an account in Slack.
```

Comment:
Can you change `here` to `chat.wordpress.org` please? This is because "Check out here for instructions" isn't grammatically correct.

Once that's resolved it's great. Thanks for the fix!